.vsts.pipelines/builds/ci-internal.yml

```diff
@@ -1,3 +1,11 @@
+trigger:
```

Comment:
I think we should consider removing the ci-internal.yml and corresponding build definitions if possible.  Same comment applies to the 3.1 PR.